website/blog/01-introducing-cmod.html

```
@@ -0,0 +1,167 @@
+          <li><strong>No standard package manager.</strong> Every team reinvents dependency management with shell scripts, Git submodules, or third-party tools that require their own ecosystems.</li>
+          <li><strong>Header-based compilation is fundamentally broken.</strong> Textual <code>#include</code> means redundant parsing, macro pollution, and non-deterministic builds. A single header change can trigger a full rebuild.</li>
+          <li><strong>Existing tools don't understand modules.</strong> C++20 introduced modules three years ago, but the tooling ecosystem never caught up. CMake treats modules as an afterthought. Conan and vcpkg are still header-first.</li>
+          <li><strong>Reproducibility is optional.</strong> Without mandatory lockfiles and pinned toolchains, "works on my machine" remains the norm.</li>
+        </ul>
+
+        <h2>What cmod Does Differently</h2>
+
+        <h3>Git Is Your Registry</h3>
+        <p>cmod doesn't require a central package server. Module identity is bound directly to Git URLs:</p>
+        <pre><code>[dependencies]
+"github.com/fmtlib/fmt" = ">=10.0"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)